cg/store/models.py

```diff
@@ -975,6 +976,18 @@ class RunDevice(Base):
     }
 
 
+class IlluminaFlowCellDevice(RunDevice):
```

Comment:
Device seems superfluous
```suggestion
class IlluminaFlowCell(RunDevice):
```

Reply:
fixed